modules/odc-installing-helm-charts.adoc

```diff
@@ -19,7 +19,15 @@ image::odc_helm_chart_devcatalog.png[]
 +
 . In the *Install Helm Chart* page:
 .. Enter a unique name for the release in the *Release Name* field.
-.. Optionally, in the YAML editor, modify the YAML file.
-. Click *Install* to create a Helm release. You will be redirected to the *Topology* view where the release is displayed. If the Helm chart has release notes, the chart is pre-selected and the right panel displays the release notes for that release.
+.. Select the required chart version from the *Chart Version* dorp-down.
++
+.Select chart version
+image::odc_helm_chart_select_chart_ver.png[]
++
+.. You can use the *Form view* of the *YAML view* to configure your Helm chart.
+.. Click *Install* to save the required changes and install the chart.
```

Comment:
I am not sure why this part of the earlier step was removed: You will be redirected to the *Topology* view where the release is displayed. If the Helm chart has release notes, the chart is pre-selected and the right panel displays the release notes for that release. 
The last sentence was considered a feature worth documenting for the last release.